README.md

```diff
@@ -133,6 +126,106 @@ jobs:
           atmos-gitops-config-path: ./.github/config/atmos-gitops.yaml    
 ```
 
+
+### Migrating from `v1` to `v2`
+
+`v2` works only with `atmos >= 1.63.0`
+
+`v2` drops `install-terraform` input because terraform is not required for affected stacks call
+
+`v2` drop `atmos-gitops-config-path` input and  `./.github/config/atmos-gitops.yaml`  config. Now you have to use GitHub actions inputs and `atmos.yaml` to specify that configuration.
```

Comment:
```suggestion

The notable changes in `v2` are:

- `v2` works only with `atmos >= 1.63.0`

- `v2` drops `install-terraform` input because terraform is not required for affected stacks call

- `v2` drops `atmos-gitops-config-path` input and the `./.github/config/atmos-gitops.yaml` config file. Now you have to use GitHub Actions environment variables to specify the location of the `atmos.yaml`.
```

Reply:
Done